scene/gui/rich_text_label.cpp

```diff
@@ -1881,7 +1910,10 @@ void RichTextLabel::_bind_methods() {
 	ADD_GROUP("BBCode", "bbcode_");
 	ADD_PROPERTY(PropertyInfo(Variant::BOOL, "bbcode_enabled"), "set_use_bbcode", "is_using_bbcode");
 	ADD_PROPERTY(PropertyInfo(Variant::STRING, "bbcode_text", PROPERTY_HINT_MULTILINE_TEXT), "set_bbcode", "get_bbcode");
+
+
```

Comment:
You might just make clang-format run on save, and put a `none` in the default rule so it doesn't mess up with other C++ projects :)